deploy/dev/5_load_environment.sh

```diff
@@ -8,9 +8,20 @@ function main() {
 
   set_namespace "$APP_NAMESPACE_NAME"
 
-  configure_secret
+  if [ "${DEV}" = "false" ]; then
+    configure_secret
 
-  deploy_env
+    deploy_env
+  else
+    setup_helm_environment
+
+    create_k8s_secret
+
+    deploy_chart
+
+    deploy_helm_app
```

Comment:
some of these methods are implemented here and some (i guess) are implemented in `utils.sh`. is there a reason for this? If not, can we have an alignment?

Reply:
because they are only referenced/needed in this file. `Utils` are for common functions across the codebase so because I only need them for one script, I declared them there. 

I moved `deploy_chart` to utils because I saw I created it twice in 2 scripts